section/models.py

```diff
@@ -303,9 +303,12 @@ def get_context(self, request, *args, **kwargs):
         return context
     
     def get_section_articles(self, order='-explicit_published_at') -> QuerySet:
+        import datetime
         # return ArticlePage.objects.from_section(section_root=self)
         # section_articles = ArticlePage.objects.live().public().filter(current_section=self.slug).order_by(order)
-        section_articles = ArticlePage.objects.live().public().descendant_of(self).order_by(order)
+        #section_articles = ArticlePage.objects.live().public().descendant_of(self).order_by(order)
+        section_articles = ArticlePage.objects.live().child_of(self).filter(explicit_published_at__gt=datetime.datetime.now()-datetime.timedelta(days=60)).order_by(order)
```

Comment:
Some suggestions to make the code more readable for future maintainers:

- Move time delta to a variable with a descriptive name
- Comment explaining the rationale behind this optimization
- Split queryset function calls on to separate lines for readability 

```suggestion
        # Optimization: limit queryset to articles published in the last 60 days to shorten query time
        recent_articles_delta = datetime.datetime.now()-datetime.timedelta(days=60)

        section_articles = ArticlePage.objects
            .live() \
            .child_of(self) \
            .filter(explicit_published_at__gt=recent_articles_delta) \
            .order_by(order)
```